### PR TITLE
Cleanup event spec and fixes #21

### LINKED ISSUE #21: Events - CupMatches
Will throw an error,  `PG::NotNullViolation: ERROR: null value in column "name" violates not-null constraint DETAIL`

### PR DESCRIPTION
**Pull request by DenniJensen**
Changed the creation of events. The `EventController` get a hash with the keyword `event`. 

Also in:
- Adds guard to project
- bundle update
- small cleanup (remove damn new lines)

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -23,9 +23,7 @@ def index
   def show
     @reply = Reply.find_or_initialize_by(event_id: @event.id,
                                          user_id: current_user.id)
-
```
Comment (elgrillo83) [inline app/controllers/events_controller.rb]: matter of style but I prefer whitespace - better readability IMHO

File: app/models/events/cup_match.rb [hunk 2]
```diff
@@ -6,11 +6,19 @@ class CupMatch < Event
...
-  def name
+  before_validation :generate_name, prepend: true
+
+  private
+
+  def generate_name
+    self.name = name_order
+  end
+
+  def name_order
```
Comment (elgrillo83) [inline app/models/events/cup_match.rb]: why extract another method (name_order)? I'd prefer self.name = if else end

File: bin/rails [hunk 3]
```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env ruby
+begin
```
Comment (elgrillo83) [inline bin/rails]: can you please explain?

File: bin/spring [hunk 4]
```diff
@@ -1,18 +1,15 @@
 #!/usr/bin/env ruby
 
-# This file loads spring without using Bundler, in order to be fast
```
Comment (elgrillo83) [inline bin/spring]: can you please explain?

File: db/schema.rb [hunk 5]
```diff
@@ -17,20 +17,22 @@
   enable_extension "plpgsql"
 
   create_table "events", force: :cascade do |t|
+    t.datetime "datetime",                      null: false
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "description",   limit: 255
```
Comment (elgrillo83) [inline db/schema.rb]: where did you the limits go?

File: bin/rails (hunk 3 above)
Comment (DenniJensen) [inline bin/rails]: This is a execution file for spring.

File: bin/spring (hunk 4 above)
Comment (DenniJensen) [inline bin/spring]: No, this was auto generated

File: db/schema.rb (hunk 5 above)
Comment (DenniJensen) [inline db/schema.rb]: Cant tell. This was generated after a migration.

**Pull request merged**